.gitignore

```diff
@@ -1,3 +1,10 @@
 /target
 .gdb_history
-.vscode
\ No newline at end of file
+.vscode
+
+# generated analysis html
+/analysis
+# python environment
+/venv
+# benchmark output json
+*.json
```

Comment:
This would match all json files including sub-dirs. I'd prefer for it to be top-level only.

Reply:
fixed by adding a leading slash